### DISCUSSION TITLE
[Enhancement]: Settings/Themes — Adjustable Accent Color

### DISCUSSION THREAD

**Discussion by SoNothingMC** [ciderapp/Cider#656] [Added Features]
### The feature you would like

A settings option to open a color picker to select a custom accent color for Cider.

### Alternatives and Other Information

Initial concept is inspired by Doppi Player: <https://doppi.app> | <https://reddit.com/r/doppi>, but this particular idea is pretty generic lol

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SmartWater2021** [comment]
Adding onto this, I think having the option to choose a different accent colour for when you have the 'podcasts' tab open would be cool.
I.e. When listening to music, the accent colour would be the normal Cider red colour, but when listening to a podcast it'd change to be the purple of apple podcasts.

**Reply by Amaru8** [reply]
**Hi @SmartWater2021!"** Please open a new discussion for your feature request on [this link](https://github.com/ciderapp/Cider/discussions/new?category=feature-request).

**Comment by Amaru8** [comment]
**Hi @SoNothingMC, I have some great news!** This feature is now implemented in [this commit](https://github.com/ciderapp/Cider/commit/d0c2d6d36ec4aa3823805e2045bb4cf1b45ea9ae) for the latest development version of **Cider**, and will be included in the next Microsoft Store update.
